model/event/event_finder.go

```diff
@@ -39,6 +39,24 @@ func Find(coll string, query db.Q) ([]EventLogEntry, error) {
 	return events, nil
 }
 
+func FindOne(query db.Q) (*EventLogEntry, error) {
+	event := &EventLogEntry{}
+	err := db.FindOneQ(AllLogCollection, query, event)
+	if adb.ResultsNotFound(err) {
+		return nil, nil
+	}
+	return event, err
+}
+
+func Aggregate(pipeline []bson.M) ([]EventLogEntry, error) {
```

Comment:
do we want to take in an  "out" or "result" interface to be more consistent with other Aggregate functions we have?

Reply:
It's generally less preferred to pass C-style in-out parameters if you can just pass back a typed value. The only reason some things accept in-out parameters like `db.Aggregate` is because it can't know the type that's being aggregated, but in this case, we know that it must be `[]EventLogEntry`.

Reply:
I guess we can just call db.Aggregate directly in the functions